Automattic/sensei
# Dev env fixes (.cache and Husky)

### Changes proposed in this Pull Request

* It [migrates the husky scripts](https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v6) that stopped working after the dependency upgrade.
* It also adds the `.cache` to our .gitignore until we're [able to upgrade](https://github.com/Automattic/sensei/pull/3578/commits/e2105dee49dc4e44d98099c9e99708a82e16c61a) and dig into the latest calypso build version.

### Testing instructions

* Add a commit, and make sure it runs the husky scripts.
* Build the package, and make sure the `.cache` is not in the git changes to be added to stage.
